cms/templates/maintenance/base.html

```diff
@@ -0,0 +1,21 @@
+<%page expression_filter="h"/>
+<%inherit file="../base.html" />
+<%def name='online_help_token()'><% return 'maintenance' %></%def>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+%>
+<%block name="content">
+<div class="wrapper-content wrapper">
+    <div class="maintenance-header">
+        <h2>
+            <a href="${reverse('maintenance:maintenance')}">
```

Comment:
I have added this in the block, so that this will be available in all child templates. All will have the link so this would too have the link so as to make templates consistent.